packages/console/src/pages/Connectors/components/ConnectorRow/index.tsx

```diff
@@ -7,31 +7,49 @@ import Status from '@/components/Status';
 import { connectorTitlePlaceHolder } from '@/consts/connectors';
 
 import ConnectorName from '../ConnectorName';
+import * as styles from './index.module.scss';
 
 type Props = {
   type: ConnectorType;
   connector?: ConnectorDTO;
+  connectors?: ConnectorDTO[];
   onClickSetup?: () => void;
 };
 
-const ConnectorRow = ({ type, connector, onClickSetup }: Props) => {
+const ConnectorRow = ({ type, connector, connectors, onClickSetup }: Props) => {
   const { t } = useTranslation(undefined);
 
   return (
     <tr>
       <td>
-        <ConnectorName type={type} connector={connector} />
+        <ConnectorName type={type} connector={connector ?? connectors?.[0]} />
```

Comment:
We display the first item in the connector group here, is it by design?

Reply:
<img width="752" alt="image" src="https://user-images.githubusercontent.com/10806653/170669597-98ae8911-b643-4e12-900a-0b5394e08667.png">

Reply:
Good call, id is now removed, and Figma updated too.